crates/solidity/testing/utils/src/version_pragmas/mod.rs

```diff
@@ -68,41 +68,43 @@ fn extract_pragma(expression_node: &Node) -> Result<VersionPragma> {
         .collect();
 
     match inner_expression.kind {
-        RuleKind::VersionPragmaBinaryExpression => match &inner_children[..] {
-            [left, operator, right] => {
-                let Node::Token(operator) = operator else {
-                    bail!("Expected rule: {operator:?}");
-                };
-
-                match operator.kind {
-                    TokenKind::BarBar => {
-                        let left = extract_pragma(left)?;
-                        let right = extract_pragma(right)?;
-
-                        Ok(VersionPragma::or(left, right))
+        RuleKind::VersionPragmaOrExpression | RuleKind::VersionPragmaRangeExpression => {
```

Comment:
nit: I suggest combining both of these matches, instead of checking again for `BarBar` or `Minus`.

Reply:
Makes sense; done in 137a1dfcc3f5b5d29006532051ad2c4d53c53e03